apisix/discovery/consul/init.lua

```diff
@@ -237,53 +208,80 @@ function _M.connect(premature, consul_server, retry_delay)
     log.info("connect consul: ", consul_server.consul_server_url,
             ", watch_result status: ", watch_result.status,
             ", watch_result.headers.index: ", watch_result.headers['X-Consul-Index'],
-            ", consul_server.index: ", consul_server.index,
-            ", consul_server: ", json_delay_encode(consul_server, true))
+            ", consul_server.index: ", consul_server.index)
```

Comment:
Would you get rid of the 8 spaces indentation in the new code?

Reply:
Deleted by mistake, Undo it